lib/pbench/server/api/resources/__init__.py

```diff
@@ -412,7 +435,7 @@ class ParamType(Enum):
     STRING = ("String", convert_string)
     USER = ("User", convert_username)
 
-    def __init__(self, name: AnyStr, convert: Callable[[AnyStr], Any]):
+    def __init__(self, name: AnyStr, convert: Callable[[Any, "Parameter"], Any]):
```

Comment:
As long as we're changing this line, do you really like `AnyStr`, or should we go with `str`?

Reply:
Most of the changes outside the code I refactored were driven by continuing annoyance at the red from `vscode`'s mysterious linter. I wasn't trying to go out of my way to pick lint, but I did (arguably inappropriately) give in a make a few easy fixes. But it didn't complain about the `name: AnyStr`, and I didn't pay any attention to it. We probably should expunge all use of `AnyStr` because we rarely use byte strings and certainly not in any of this code.

Reply:
I'm right there with you on the IDE linter thing -- the sidebar is so full of "look at me!" marks that I can't always find the marks that I really care about...so, I think picking lint is a good thing...but we need to take some care that it doesn't distract from the change at hand (i.e., it would be good to segregate lint-picking to its own commit).

As for purgining `AnyStr`'s, I agree completely, but I'm inclined to attack the problem piecemeal, and this one looked like it could be changed with little impact.  😉